CONTRIBUTING.md

```diff
@@ -14,78 +14,127 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-## Contributing to AMP HTML
+# Contributing to AMP HTML
 
 The AMP HTML project strongly encourages technical [contributions](https://www.ampproject.org/contribute/)!
 
 We hope you'll become an ongoing participant in our open source community but we also welcome one-off contributions for the issues you're particularly passionate about.
 
-### Filing Issues
+## Filing issues
 
-**Suggestions**
+### Suggestions
 
-The AMP HTML project is meant to evolve with feedback - the project and its users greatly appreciate any thoughts on ways to improve the design or features. Please use the `enhancement` tag to specifically denote issues that are suggestions - this helps us triage and respond appropriately.
+The AMP Project is meant to evolve with feedback.  The project and its users appreciate your thoughts on ways to improve the design or features.
 
-**Bugs**
+To make a suggestion [file an issue](https://github.com/ampproject/amphtml/issues/new) and add the [Type: Feature Request](https://github.com/ampproject/amphtml/labels/Type%3A%20Feature%20Request) label.
```

Comment:
I went ahead and removed adding the label altogether to avoid confusing it.  I'm assuming we'll be able to tell from context that it's an Intent to Implement and someone with permission to do so will add the tag (onduty, etc.)